### PR TITLE
Update Perses Header 

### PR DESCRIPTION
**Pull request by eunicorn**
This PR just makes the dashboard toolbar looks nicer :) 

- moved TemplateVariableList into DashboardToolbar

Before: 
image

After:
image

### CODE REVIEW

**Review by LukeTillman — APPROVED** [review]
Looks great!

File: ui/dashboards/src/components/DashboardToolbar.tsx [hunk 1]
```diff
@@ -36,47 +38,69 @@ export const DashboardToolbar = (props: DashboardToolbarProps) => {
...
+                </Button>
+              </Stack>
+            </Box>
+          </Box>
+          <Box
+            sx={{
+              display: 'flex',
+              width: '100%',
+              padding: (theme) => theme.spacing(2),
+              transition: 'top 1s cubic-bezier(0.17, 0.04, 0.03, 0.94)',
```
Comment (cndonovan) [inline ui/dashboards/src/components/DashboardToolbar.tsx:65]: Ooo does this just apply as the Box gets rendered

File: ui/dashboards/src/components/DashboardToolbar.tsx [hunk 2]
```diff
@@ -36,47 +38,69 @@ export const DashboardToolbar = (props: DashboardToolbarProps) => {
...
+    <Toolbar
+      disableGutters
+      sx={{
+        display: 'block',
+      }}
+    >
       {isEditMode ? (
-        <>
+        <Stack spacing={2}>
+          <Box sx={{ backgroundColor: (theme) => theme.palette.primary.light + '20' }}>
```
Comment (cndonovan) [inline ui/dashboards/src/components/DashboardToolbar.tsx:49]: What does `+ 20` do?

File: ui/dashboards/src/components/DashboardToolbar.tsx (hunk 1 above)
Comment (eunicorn) [reply ui/dashboards/src/components/DashboardToolbar.tsx]: no haha i was playing around with animations and forgot to remove it :P

**Review by cndonovan — APPROVED** [review]
beautifullll

**Review by sjcobb — APPROVED** [review]
🚀

**Pull request merged**